src/utils/languages.ts

```diff
@@ -34,9 +35,31 @@ export const availableLanguages: Language[] = [
   { code: 'fi', name: 'Suomi', flag: '🇫🇮' },
   { code: 'no', name: 'Norsk', flag: '🇳🇴' },
   { code: 'tr', name: 'TÜRKİSH', flag: '🇹🇷' },
-  { code: "pl", name: "Polski", flag: "🇵🇱" },
+  { code: 'pl', name: 'Polski', flag: '🇵🇱' },
   { code: 'zhtw', name: '中文(台灣)', flag: '🇹🇼' },
   { code: 'bg', name: 'бълежи', flag: '🇧🇬' },
+  { code: 'nl', name: 'Nederlands', flag: '🇳🇱' },
+  { code: 'sk', name: 'Slovenčina', flag: '🇸🇰' },
+  { code: 'ro', name: 'Română', flag: '🇷🇴' },
+  { code: 'lv', name: 'Latviešu', flag: '🇱🇻' },
+  { code: 'et', name: 'Eesti', flag: '🇪🇪' },
+  { code: 'hr', name: 'Hrvatski', flag: '🇭🇷' },
+  { code: 'sl', name: 'Slovenščina', flag: '🇸🇮' },
+  { code: 'el', name: 'Ελληνικά', flag: '🇬🇷' },
+  { code: 'ru', name: 'Русский', flag: '🇷🇺' },
+  { code: 'uk', name: 'Українська', flag: '🇺🇦' },
```

Comment:
duplication

https://github.com/miurla/babyagi-ui/pull/57/files#diff-6703b91dc8242fb5425dfa1a282a7ea5c9a9bc92cd8f453a61aa06dc6b2eb5b0R25

Reply:
Ohh, I found an other one not uk 😅 It duplicated because I force pushed it.

Reply:
Thx!